packages/cli/src/credentials/credentials.controller.ts

```diff
@@ -198,7 +198,7 @@ export class CredentialsController {
 			throw new BadRequestError('Managed credentials cannot be updated');
 		}
 
-		const decryptedData = this.credentialsService.decrypt(credential);
+		const decryptedData = this.credentialsService.decrypt(credential, true);
```

Comment:
in the long run we need to rethink this approach of explicitly redacting or not, since it's prone to people forgetting to pass in this parameter.
We need some mechanism to ensure that decrypted secrets can **never** reach the frontend, and only the execution engine can read these secrets.

Reply:
I'm a fan of having DTOs for outgoing data as well. Comes at a performance cost though, but I think we have bigger bottlenecks.